interop-testing/src/generated/main/grpc/io/grpc/testing/integration/UnimplementedServiceGrpc.java

```diff
@@ -53,6 +54,19 @@ public void unimplementedCall(com.google.protobuf.EmptyProtos.Empty request,
         io.grpc.stub.StreamObserver<com.google.protobuf.EmptyProtos.Empty> responseObserver);
   }
 
+  public static abstract class AbstractUnimplementedService implements UnimplementedService, io.grpc.stub.BindableService {
+
+    @java.lang.Override
+    public void unimplementedCall(com.google.protobuf.EmptyProtos.Empty request,
+        io.grpc.stub.StreamObserver<com.google.protobuf.EmptyProtos.Empty> responseObserver) {
+      unimplementedStreamResponse(responseObserver);
+    }
+
+    @Override public io.grpc.ServerServiceDefinition bindService() {
```

Comment:
Doh. We missed in the previous PR this should be `@java.lang.Override` to avoid (unlikely) name collision. That can be a follow-up fix.